code/script.js

```diff
@@ -237,107 +241,189 @@ const generateBoard = () => {
         <img src=${person.img} alt=${person.name}>
         <div class="guess">
           <span>Guess on ${person.name}?</span>
-          <button class="filled-button small" onclick="guess('${person.name}')">Guess</button>
+          <button class="filled-button small" id='guess-who' onclick="guess('${person.name}')">Guess</button>
```

Comment:
Could use camelCase for the ID.

Reply:
`id` is a CSS selector so according to Guidelines it should be in kebab-case, and the pointer to it created in the javascript should be stored in camelCase variable.